src/osemgrep/reporting/Cli_json_output.ml

```diff
@@ -389,6 +389,7 @@ let cli_match_of_core_match (hrules : Rule.hrules) (m : Out.core_match) :
             rule_message
       in
       (* message where the metavars have been interpolated *)
+      (* TODO(secrets): apply masking logic here *)
```

Comment:
When do you plan to port it to osemgrep?

Reply:
Trying to find a good point to slot this in. I think I may be able late this week/early next.